ts_src/psbt.ts

```diff
@@ -798,6 +804,14 @@ export class Psbt {
       c.__TX.setOutputSurjectionProof(outputIndex, surjectionProof);
     });
 
+    range(numOutputs)
+      .filter(index => !outputsIndexToBlind!.includes(index))
+      .forEach(unconfidentialIndex => {
+        c.__TX.setOutputNonce(unconfidentialIndex, Buffer.alloc(0))
+        // c.__TX.setOutputRangeProof(unconfidentialIndex, Buffer.alloc(0))
+        // c.__TX.setOutputSurjectionProof(unconfidentialIndex, Buffer.alloc(0))
+      })
+
```

Comment:
what is this?

Reply:
I was thinking that it will solve the problem, let's remove it :)